tests/user_examples/extra_src/shared/general_continuum/riemann_solver_extra.h

```diff
@@ -11,16 +11,8 @@ namespace SPH
 		template <class FluidI, class FluidJ>
 		AcousticRiemannSolverExtra(FluidI& fluid_i, FluidJ& fluid_j)
```

Comment:
Here, the "extra" meaning is not clear. Please try to figure out a term to present the difference between this and the original AcousticRiemannSOlver.

Reply:
Here, I just change the coefficient in the Riemann solver for my cases.
https://github.com/Xiangyu-Hu/SPHinXsys/blob/94e29bd5920bbe6d5915ef2fd1589fa72ce44ec5/tests/user_examples/extra_src/shared/general_continuum/riemann_solver_extra.cpp#L7
The coefficient is changed to `(Real)Dimensions * Real(20)`.
Maybe we can set this coefficient as a input parameter in the original Riemann solver later.